src/kernel/accumulate_maxsumexp/cpu.cc

```diff
@@ -32,30 +32,34 @@ void cpu(Index nelems, const T* src_, T* dst_)
  * @param[inout] dst_: Destination of the maxsumexp accumulation
  * */
 {
-    using Y = typename CPUComputeType<T>::value;
-    constexpr Y zero{0.0};
-    auto src = reinterpret_cast<const Y *>(src_);
-    auto dst = reinterpret_cast<Y *>(dst_);
+    // using Y = typename CPUComputeType<T>::value;
```

Comment:
remove commented out code; add using for the T::repr_t type to simplify static casts